tests/sentinel/tests/05-manual.tcl

```diff
@@ -1,18 +1,26 @@
 # Test manual failover
 
-source "../tests/includes/init-tests.tcl"
+proc test_sentinel_failover {type master_id} {
 
 foreach_sentinel_id id {
     S $id sentinel debug info-period 2000
     S $id sentinel debug default-down-after 6000
     S $id sentinel debug publish-period 1000
 }
 
-test "Manual failover works" {
+set val 0
+
+test "Manual failover works - $type" {
+    R $master_id del foo
     set old_port [RPort $master_id]
     set addr [S 0 SENTINEL GET-PRIMARY-ADDR-BY-NAME mymaster]
     assert {[lindex $addr 1] == $old_port}
 
+    # Rename the FAILOVER command so that we can fallback to REPLICAOF NO ONE.
+    if {$type == "legacy"} {
+        S 0 SENTINEL SET mymaster rename-command FAILOVER NON-EXISTENT
```

Comment:
In real production, this step: call rename-command explicitly is must?

Reply:
No, it is just a test step that try to cover the fallback logic.

Reply:
Let's explain in the comment

    # We simulate a server that doesn't have the FAILOVER command.